demo/demoserver.py

```diff
@@ -0,0 +1,64 @@
+'''Demoserver for RobotFlow
+'''
+from os.path import abspath
+import cherrypy
+from pathlib2 import Path
+
+
+class FormGenerator():
+    '''Main CherryPy class for showing pages
+    '''
+    def __init__(self):
+        self.valid_customers = ['345', '732', '834']
+        self.expired_customers = ['432', '545', '678']
+
+    @cherrypy.expose
+    def index(self):
+        '''Display main page
+        '''
+        # pylint: disable=no-self-use
+        return open("form/form.html")
+
+    @cherrypy.expose
+    def generate(self, **kwargs):
+        '''Show customerId status
+        '''
+        keywords = kwargs.keys()
+        if 'customerid' not in keywords:
+            raise Exception
+        customerid = kwargs['customerid']
+        result = ''
+        try:
+            if len(customerid) == 3:
+                if customerid == '000':
+                    raise cherrypy.HTTPError(500)
```

Comment:
To be very strict, error code 400 might be better suited, 500 is more for an unhandled server error, 400 suits better for client giving invalid data. But in the end in this context that's not big issue as this is not real server code.